src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -74,6 +74,13 @@ Meta information and execution details about the current vu and iteration.
 | iterationInScenario | integer | The identifier of the iteration in the current scenario. |
 | idInInstance        | integer | The identifier of the VU across the instance. |
 | idInTest            | integer | The globally unique (across the whole test run) identifier of the VU. |
+| tags                | object  | The map that gives control over VU's Tags. The Tags will be included in every metric emitted by the VU and the Tags' state is maintained across different iterations of the same Scenario while the VU exists. |
```

Comment:
Suggestions: 

1 - Add a link to the example below.

2 - Add a link to k6 Tags for further reference. It could be something like: 

Read more about [k6 Tags](https://k6.io/docs/using-k6/tags-and-groups/#tags) and check out the [example](#the-links-to-example).